src/kem/sike/CMakeLists.txt

```diff
@@ -92,7 +96,8 @@ if(OQS_ENABLE_KEM_sike_p434 OR
             if(OQS_ENABLE_KEM_sike_p751 OR OQS_ENABLE_KEM_sidh_p751 OR OQS_ENABLE_KEM_sike_p751_compressed OR OQS_ENABLE_KEM_sidh_p751_compressed)
                 target_compile_definitions(sike PRIVATE USE_SIKEP751_ASM)
             endif()
-            if(OQS_USE_ADX_INSTRUCTIONS AND NOT OQS_DIST_BUILD)
+            if(OQS_USE_BMI2_INSTRUCTIONS)
```

Comment:
The switch from checking for ADX to BMI2 I understand (if the latter also includes/implies the former or the code builds regardless of ADX presence) but leaving away `OQS_DIST_BUILD` here seems to imply that the resultant code will work regardless of CPU features found, no? Asked another way, what code difference results for SIKE with `OQS_DIST_BUILD` ON or OFF?

Reply:
The OQS_USE_XXX_INSTRUCTIONS are only set when OQS_DIST_BUILD=OFF.

https://github.com/open-quantum-safe/liboqs/blob/efd7b1926ac485bd2c284986cb94e51f4d50450c/.CMake/compiler_opts.cmake#L51-L53

Reply:
OK, I was confused how then run-time feature detection works when OQS_DIST_BUILD=ON given this (example Kyber, other places equivalent) code: https://github.com/open-quantum-safe/liboqs/blob/efd7b1926ac485bd2c284986cb94e51f4d50450c/src/kem/kyber/kem_kyber_512.c#L44-L56 . But this https://github.com/open-quantum-safe/liboqs/blob/efd7b1926ac485bd2c284986cb94e51f4d50450c/.CMake/alg_support.cmake#L178-L180 in combination with this https://github.com/open-quantum-safe/liboqs/blob/efd7b1926ac485bd2c284986cb94e51f4d50450c/CMakeLists.txt#L33-L35 makes it clear again. So case closed. Thanks.